src/bridge.js

```diff
@@ -0,0 +1,29 @@
+const Bridge = require('matrix-appservice-bridge').Bridge;
+const {startSMTP}= require('./email');
```

Comment:
Might be worth getting a linter rule for consistent spacing in brackets.

```suggestion
const { startSMTP }= require('./email');
```